usbmon.c

```diff
@@ -89,32 +90,46 @@ int main(int argc, char **argv) {
     struct collectd cv;
     struct utsname u;
     const char *path, *usbpath, *vendor, *serial, *speed, *action;
-    int fd, ret, co = 0;
+    int fd, ret, co = 0, nomon = 0, opt, tv_sec = 10;
```

Comment:
also here do not call it tv_sec rather, interval

Reply:
changed it .